packaging_docker/testnet/azure/secret-node-azure-template/azuredeploy.json

```diff
@@ -127,7 +127,7 @@
       "type": "Microsoft.Network/networkSecurityGroups",
       "apiVersion": "2020-04-01",
       "name": "[variables('networkSecurityGroupName')]",
-      "location": "uksouth",
+      "location": "[parameters('location')]",
```

Comment:
where does this come from?

Reply:
The parameters field in the top level. Those are the values you accept from the user etc.